src/cuga/backend/cuga_graph/nodes/cuga_supervisor/nodes/prepare_agents_and_prompt.py

```
@@ -222,28 +233,30 @@ async def prepare_agents_and_prompt(
-            enable_todos=True,
+            enable_todos=enable_todos,
             special_instructions=effective_special_instructions,
         )
 
+        update_payload = {
+            "tools_prepared": True,
+            "prepared_prompt": dynamic_prompt,
+            "step_count": 0,
+            "available_agents": {
+                name: AgentInfo(name=name, type=info["type"], description=info["description"]).model_dump()
+                for name, info in zip([a["name"] for a in agent_list], agent_list)
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)